scio-test/src/main/scala/com/spotify/scio/testing/SCollectionMatchers.scala

```diff
@@ -35,12 +35,65 @@ import org.hamcrest.MatcherAssert.assertThat
 import scala.collection.JavaConverters._
 import scala.reflect.ClassTag
 import com.twitter.chill.ClosureCleaner
+import cats.kernel.Eq
+import org.apache.beam.sdk.testing.SerializableMatchers
+import com.spotify.scio.coders.CoderMaterializer
+import org.apache.beam.sdk.options.PipelineOptionsFactory
+import com.spotify.scio.options.ScioOptions
+
+private[testing] case class TestWrapper[T: Eq](value: T) {
+  override def toString: String = Pretty.printer.apply(value).render
+  override def equals(other: Any): Boolean =
+    other match {
+      case TestWrapper(o: T) => Eq[T].eqv(value, o)
+      case o                 => value.equals(o)
+    }
+}
+
+object TestWrapper {
+  def wrap[T: Coder: Eq](coll: SCollection[T]) =
+    coll.map(t => TestWrapper(t))
+
+  implicit def testWrapperCoder[T: Coder: Eq]: Coder[TestWrapper[T]] =
+    Coder.gen[TestWrapper[T]]
+}
+
+private object ScioMatchers {
+  private def supplierFromCoder[A: Coder, B](@transient a: A)(builder: A => B) = {
+    val options = PipelineOptionsFactory.create().as(classOf[ScioOptions])
+    options.setNullableCoders(true)
```

Comment:
@jto can you add a comment on why do we need to set this?

Reply:
Good catch! It was not necessary. Fixed it :)